HunterPie.UI/Overlay/Widgets/Damage/View/MeterView.xaml

```diff
@@ -55,6 +56,16 @@
                                FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
                                VerticalAlignment="Center"
                                FontSize="13"/>
+                    <TextBlock Foreground="{StaticResource HUNTERPIE_FOREGROUND_PRIMARY}"
+                               FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
+                               VerticalAlignment="Center"
+                               FontSize="13"
+                               Visibility="{Binding MaxDeaths, Converter={StaticResource MaxDeathsToVisibilityConverter}}">
+                        <Run d:Text=" /" 
+                             Text=" /"/>
+                        <Run d:Text="3"
+                             Text="{Binding MaxDeaths, Mode=OneWay}"/>
+                    </TextBlock>
```

Comment:
Use the deaths `TextBlock` for that, you can replace this whole block:

https://github.com/HunterPie/HunterPie/blob/5243cd5ee0deaeb37de8ff936bac3355b47daeab/HunterPie.UI/Overlay/Widgets/Damage/View/MeterView.xaml#L53-L68

With this:

```xaml
                    <TextBlock Foreground="{StaticResource HUNTERPIE_FOREGROUND_PRIMARY}"
                               FontFamily="{StaticResource HUNTERPIE_DEFAULT_FONT}"
                               VerticalAlignment="Center"
                               FontSize="13">
                        <Run d:Text="2"
                             Text="{Binding Deaths, Mode=OneWay}"/>
                        <Run Text="/"
                             Visibility="{Binding MaxDeaths, Converter={StaticResource MaxDeathsToVisibilityConverter}}"/>
                        <Run d:Text="3"
                             Text="{Binding MaxDeaths, Mode=OneWay}"
                             Visibility="{Binding MaxDeaths, Converter={StaticResource MaxDeathsToVisibilityConverter}}"/>
                    </TextBlock>
```

Reply:
Run.Text has no visibility property....